files/angular-material-icons/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",
+  "name": "angular-material-icons",
+  "repo": "klarsys/angular-material-icons",
+  "files": {
+    "include": ["angular-material-icons.js"]
```

Comment:
Did you not mean to include this file?
https://github.com/klarsys/angular-material-icons/blob/master/angular-material-icons.min.js